script.js

```diff
@@ -97,10 +97,99 @@ class CircleMover extends Mover {
 	}
 }
 
+class Draggable {
+	constructor(x,y,w,h){
+		this.dragging = false;
+		this.rollover = false;
```

Comment:
What does `rollover` mean? A more descriptive name might be helpful if you can think of one.

Reply:
Changed to mouseOver as it's for changing come colors when you are moused over an endpoint.